_studio/mfx_lib/decode/av1/src/mfx_av1_dec_decode.cpp

```diff
@@ -988,6 +988,11 @@ mfxStatus VideoDECODEAV1::SubmitFrame(mfxBitstream* bs, mfxFrameSurface1* surfac
                 return MFX_ERR_NONE;
             }
 
+            if (sts == MFX_ERR_NONE && umcRes != UMC::UMC_OK)
```

Comment:
Actually, it seems this can actually be easily satisfied with for example UMC::UMC_ERR_NOT_ENOUGH_DATA....

Reply:
In fact, no, in case of `UMC::UMC_ERR_NOT_ENOUGH_DATA` sts is set to `MFX_ERR_MORE_DATA` so this condition will not be satisfied.
https://github.com/Intel-Media-SDK/MediaSDK/blob/6b6cc790c08426b4111094fa311dd34e2e4541fb/_studio/mfx_lib/decode/av1/src/mfx_av1_dec_decode.cpp#L954-L959